src/config/index.ts

```diff
@@ -31,6 +29,15 @@ export const REFERENCE = "OverseasEntitiesReference";
 export const PAYMENT_REQUIRED_HEADER = "x-payment-required";
 export const PAYMENT_PAID = "paid";
 
+//  FEATURE FLAGS
+const envFlagSfx = 'FEATURE_FLAG_';
```

Comment:
I think this represents a prefix not a suffix (if that's what 'Sfx' is meant to represent)?

Suggest changing name to `featureFlagNamePrefix` if this remains.

Reply:
fixed